Firebase/android/app/src/main/java/com/novoda/bonfire/link/FirebaseDynamicLinkFactory.java

```diff
@@ -0,0 +1,43 @@
+package com.novoda.bonfire.link;
+
+import android.net.Uri;
+
+import com.novoda.bonfire.user.data.model.User;
+
+import java.net.URI;
+
+public class FirebaseDynamicLinkFactory implements LinkFactory {
```

Comment:
We called the interface `DynamicLinkFactory`. I mean, it doesn't matter that much, but just thought I'd point it out.